zhaquirks/tuya/tuya_illuminance.py

```diff
@@ -0,0 +1,31 @@
+"""Tuya illuminance sensors."""
+
+from zigpy.types import t
+
+from zhaquirks.tuya.builder import TuyaQuirkBuilder
+
+
+class BrightnessLevel(t.enum8):
+    """Brightness level enum."""
+
+    Low = 0x00
+    Medium = 0x01
+    High = 0x02
+    Strong = 0x03
+
+
+(
+    TuyaQuirkBuilder("_TZE200_khx7nnka", "TS0601")  # Tuya XFY-CGQ-ZIGB
+    .applies_to("_TZE204_khx7nnka", "TS0601")
+    .applies_to("_TZE200_yi4jtqq1", "TS0601")
+    .tuya_enum(
+        dp_id=1,
+        attribute_name="brightness_level",
+        enum_class=BrightnessLevel,
+        translation_key="brightness_level",
+        fallback_name="Brightness level",
```

Comment:
If this describes the sensitivity, do we wanna do "Sensitivity level" or "Brightness sensitivity" instead?
```suggestion
        fallback_name="Brightness sensitivity",
```

Reply:
Is that what it is? z2m just lists it as `brightness_state`, although it is a `STATE_SET`. Thought maybe it had a display or something. If it is sensitivity then, yes, I agree on the name change.

Reply:
Misread that, it's another indication of brightness.